src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -709,51 +716,8 @@ void ClusterTaskManager::FillResourceUsage(
 
   int num_reported = 0;
 
-  // 1-CPU optimization
```

Comment:
Why the removal?

Reply:
There isn't really a canonical task shape any more once function descriptors are added to the scheduling class. Also, in practice, I think we always send all resource shapes with every request, so it shouldn't lead to any regressions.

Reply:
I see, I've lost the context for this part of the code. But as long as we have regression tests, I'm fine with it.